internal/formatter/backup_policy_list.go

```diff
@@ -27,18 +27,19 @@ import (
 )
 
 const (
-	defaultBackupPolicyListing = "table {{.TimeInterval}}\t{{.DaysOfTheWeek}}\t{{.BackupStartTime}}\t{{.RetentionPeriod}}\t{{.State}}"
-	timeIntervalHeader         = "Time Interval(days)"
-	daysOfTheWeekHeader        = "Days of the Week"
-	backupStartTimeHeader      = "Backup Start Time"
-	retentionPeriodInDays      = "Retention Period(days)"
-	state                      = "State"
+	defaultBackupPolicyListing    = "table {{.TimeInterval}}\t{{.IncrementalTimeInterval}}\t{{.DaysOfTheWeek}}\t{{.BackupStartTime}}\t{{.RetentionPeriod}}\t{{.State}}"
+	timeIntervalHeader            = "Time Interval(days)"
+	incrementalTimeIntervalHeader = "Incremental Time Interval(minutes)"
```

Comment:
This seems long. Maybe drop `Time` and use `mins` instead of `minutes`?

Reply:
+1 on small screen could look multiline table..

Reply:
Maybe even shorten to `Incr.`